account_wallet/wizards/wizard_account_move_refund_wallet.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="account_move_refund_wallet_view_form" model="ir.ui.view">
+        <field name="name">wizard.account_move_refund.wallet.form</field>
+        <field name="model">wizard.account_move_refund.wallet</field>
+        <field name="arch" type="xml">
+            <form>
```

Comment:
In a wizard, a little text in order to help user to know exactly what this would be done could be great.

(Add a div)